src/Apps/Collections/CollectionsApp.tsx

```diff
@@ -28,13 +28,22 @@ export class CollectionsApp extends Component<CollectionsAppProps> {
             justifyContent="space-between"
             alignItems="flex-end"
           >
-            <Serif size="8">Collections</Serif>
+            <Serif size="8">
+              <h1 className="title">Collections</h1>
```

Comment:
is the className used to set some styles here? if that's the case I would rather use styled-components.

```javascript
const Title = styled.h1`
  ...
`
```

Reply:
added for SEO based on Nicolette's recommendation here, not used for actual syles - https://artsyproduct.atlassian.net/browse/GROW-1004

Reply:
maybe it doesn't actually need a class though, now that i look closer